app/serializers/rest/admin/account_serializer.rb

```diff
@@ -2,13 +2,14 @@
 
 class REST::Admin::AccountSerializer < ActiveModel::Serializer
   attributes :id, :username, :domain, :created_at,
-             :email, :ip, :role, :confirmed, :suspended,
+             :email, :role, :confirmed, :suspended,
```

Comment:
As to not break compatibility with clients without a transition period, I'd return the latest IP in `ip` instead of deleting it.

Reply:
Are there any apps that actually use and rely on this API? I am not aware of any which is why I opted to skip it

Reply:
Good question. I don't actually know :/

Reply:
But in any case, it's not a big deal to just keep 3 lines of code to not break compatibility.